scipy/optimize/_simulated_annealing.py

```diff
@@ -0,0 +1,557 @@
+# Simulated Dual Annealing implementation.
+# Copyright (c) 2018 Sylvain Gubian <sylvain.gubian@pmi.com>,
```

Comment:
Should this copyright statement be here?  (I'm just asking, I have no clue).

Reply:
I thought it was ok as I referred to an [open source guide](https://opensource.guide/) and [this one](http://mg.readthedocs.io/python.html) but if you think this should be differently, I would be happy to adjust.